schemas/v3.1/schema.yaml

```
@@ -0,0 +1,810 @@
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+      Schema:
+        $anchor: Schema
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: 'https://json-schema.org/draft/2020-12/schema'
+              - $ref: '#SchemaExtensions'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+        $defs:
+          Extensions:
+            $anchor: SchemaExtensions
+            properties:
+              discriminator:
+                $ref: '#Discriminator'
+              externalDocs:
+                $ref: '#ExternalDocumentation'
+              xml:
+                $ref: '#XML'
+          Discriminator:
+            $anchor: Discriminator
+            type: object
+            required:
+              - propertyName
+            properties:
+              propertyName:
+                type: string
+              mapping:
+                type: object
+                additionalProperties:
+                  type: string
+          XML:
+            $anchor: XML
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+              - type: object
+                properties:
+                  name:
+                    type: string
+                  namespace:
+                    type: string
+                    format: url
+                  prefix:
+                    type: string
+                  attribute:
+                    type: boolean
+                    default: false
+                  wrapped:
+                    type: boolean
+                    default: false
+            additionalProperties: false
+      Components:
+        $anchor: Components
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+          - additionalProperties:
+              $comment: All sub-objects have the same property name constraints
+              type: object
+              propertyNames:
+                pattern: '^[a-zA-Z0-9\.\-_]+$'
+        properties:
+          schemas:
+            additionalProperties:
+              $ref: '#Schema'
+          responses:
+            additionalProperties:
+              $ref: '#Responses'
+          parameters:
+            additionalProperties:
+              $ref: '#Parameter'
+          pathItems:
+            additionalProperties:
+              $ref: '#PathItem'
+          examples:
+            additionalProperties:
+              $ref: '#Example'
+          requestBodies:
+            additionalProperties:
+              $ref: '#RequestBody'
+          headers:
+            additionalProperties:
+              $ref: '#Header'
+          links:
+            additionalProperties:
+              $ref: '#Link'
+          callbacks:
+            additionalProperties:
+              $ref: '#Callback'
+        unevaluatedProperties: true
```

```suggestion
        unevaluatedProperties: false
```

`components` is already `Extensible`, but additional `components` "containers" are not permitted.